test/lib/test-reporter.mjs

```diff
@@ -95,6 +97,15 @@ class Tracker extends Map {
   }
 }
 
+function calcTime(start, end) {
+  const delta = Number(end - start) / 1_000_000
+  return delta.toLocaleString('en-US', {
```

Comment:
I don't think this is correct.  For example:

```sh
node test/integration/agent/multiple.test.js
```
outputs:

```sh
✔ Multiple require("newrelic") (117.989167ms)
ℹ tests 1
ℹ suites 0
ℹ pass 1
ℹ fail 0
ℹ cancelled 0
ℹ skipped 0
ℹ todo 0
ℹ duration_ms 221.337959
```

But the reporter shows it as

```sh
passed: /Users/revans/code/pull-requests/node-newrelic/test/integration/agent/multiple.test.js (117,900 ms)
```

Reply:
@bizob2828 check the locale setting(s) on your system:

```
❯ node --test-reporter ./test/lib/test-reporter.mjs test/integration/agent/multiple.test.js                                 
passed: /Users/jsumners/Projects/team-repos/node-newrelic/test/integration/agent/multiple.test.js (0.2296 ms)
```

What you're seeing is a European decimal separator.